.github/workflows/release.yml

```
@@ -0,0 +1,22 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    environment: release
+
+    permissions:
+      id-token: write # IMPORTANT: this permission is mandatory for trusted publishing
+      contents: write # IMPORTANT: this permission is required for `rake release` to push the release tag
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: ruby/setup-ruby@a4effe49ee8ee5b8b5091268c473a4628afb5651 # v1.245.0
+        with:
+          ruby-version: 3.4.4
```

【nits】
```suggestion
          ruby-version: 3.4.4 # Latest stable Ruby for release builds
```

最新の安定版として3.4.4 を使用していますが、その理由がわかるコメントがあっても良いのかなと思いました。